src/test/cypress/integration/quiz-exercise/QuizExerciseAssessment.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { artemis } from '../../support/ArtemisTesting';
+import shortAnswerQuizTemplate from '../../fixtures/quiz_exercise_fixtures/shortAnswerQuiz_template.json';
+import multipleChoiceQuizTemplate from '../../fixtures/quiz_exercise_fixtures/multipleChoiceQuiz_template.json';
+
+// Accounts
+const admin = artemis.users.getAdmin();
+const tutor = artemis.users.getTutor();
+const student = artemis.users.getStudentOne();
+
+// Requests
+const courseManagementRequest = artemis.requests.courseManagement;
+
+// Common primitives
+let course: any;
+let quizExercise: any;
+
+describe('Quiz Exercise Assessment', () => {
+    before('Set up course', () => {
+        cy.login(admin);
+        courseManagementRequest.createCourse().then((response) => {
+            course = response.body;
+            courseManagementRequest.addStudentToCourse(course.id, student.username);
+            courseManagementRequest.addTutorToCourse(course, tutor);
+        });
+    });
+
+    afterEach('Delete Quiz', () => {
+        deleteQuiz();
+    });
+
+    after('Delete Course', () => {
+        cy.login(admin);
+        courseManagementRequest.deleteCourse(course.id);
+    });
+
+    describe('MC Quiz assessment', () => {
+        before('Creates a quiz and a submission', () => {
+            createQuiz();
+        });
+
+        it('Assesses a mc quiz submission automatically', () => {
+            cy.login(student);
+            courseManagementRequest.startExerciseParticipation(course.id, quizExercise.id);
+            courseManagementRequest.createMultipleChoiceSubmission(quizExercise, [0, 2]);
+            cy.wait(5000);
```

Comment:
Why is this wait needed?

Reply:
This wait is needed because the result for the submission is generated automatically by the server and that takes a while for some reason. If i visit the result page immediately after creating the submission it just says that there is no assessment yet and it also doesnt update when the result is available. So instead i just give the server 5 seconds to get the result ready and then visit the page.

Reply:
I dislike static waits. If the testserver is slow for whatever reason and doesn't provide the assessment in those 5 seconds the test would still fail. It might be worth looking into the [waitUntil plugin](https://github.com/NoriSte/cypress-wait-until).
I found an interesting example, which constantly refreshes the page until an element is found:
![image](https://user-images.githubusercontent.com/12861668/133939387-9160a364-ec15-496d-8228-807ef7a0fbe3.png)
Apparently one can also set intervals when the page should be refreshed and a timeout. This could be quite useful in this case

Reply:
So your'e telling me you want me to install a plugin so i can save myself a potential 1.5 seconds off of my 5 second wait? i understand that 1.5 seconds over thousands of runs might be something but considering there are a lot of instances in our tests where we wait for way longer than that this just seems like nitpicking. If you want to play around with this feel free to open a follow up PR.

Reply:
This does not work. We have discussed this in PMs.

Reply:
I looked a little bit into it. I came up with a method that worked for me locally. I added the function as a suggestion below.